xml/obs_maintenance_setup.xml

```diff
@@ -48,20 +51,47 @@
   <itemizedlist>
    <listitem>
     <para> openSUSE:11.4 is the <xref linkend="obs.glos.ga_project"/> in
-     this example. Frozen and not changing anymore. </para>
+     this example. It is locked and not changing anymore. </para>
    </listitem>
    <listitem>
     <para> openSUSE:11.4:Update is the <xref
-      linkend="obs.glos.update_project"/> to release official updates.
-    </para>
+      linkend="obs.glos.update_project"/> to release official updates for the
+      locked openSUSE:11.4 project.
+      Thus it links to the openSUSE:11.4 project, inheriting all packages from there. </para>
```

Comment:
Can we have "inherit all package sources" here to avoid confusion that we don't duplicate the binaries?